packages/video_player_videohole/README.md

```diff
@@ -0,0 +1,121 @@
+# video_player_videohole
+
+A video_player_videohole flutter plugin only for Tizen TV devices, supports play DRM(playready & widevine).
+
+## Required privileges
+
+To use this plugin in a Tizen application, you may need to add the following privileges in your `tizen-manifest.xml` file.
+
+```xml
+<privileges>
+  <privilege>http://tizen.org/privilege/mediastorage</privilege>
+  <privilege>http://tizen.org/privilege/externalstorage</privilege>
+  <privilege>http://tizen.org/privilege/internet</privilege>
+  <privilege>http://tizen.org/privilege/network.get</privilege>
+  <privilege>http://tizen.org/privilege/download</privilege>
+  <privilege>http://tizen.org/privilege/push</privilege>
```

Comment:
Are these three privileges necessary? They are not explicitly mentioned below.

Reply:
`network.get` and `push` are unnecessary.
But I'm not sure whether `download` will be used, it is used to allow the application to manage HTTP downloads. Maybe request drm license will need it.

Reply:
> Maybe request drm license will need it.

Does it? As far as I can see in Tizen Docs, only the [Content Download](https://docs.tizen.org/application/native/api/iot-headed/5.5/group__CAPI__WEB__DOWNLOAD__MODULE.html) API requires the download privilege, and the privilege is not part of [non API-bound privileges](https://docs.tizen.org/application/native/tutorials/details/sec-privileges/#non-api-bound-privileges).

Reply:
Okay, I know. Then you can remove the download privilege.